src/main/kotlin/com/grepp/nbe1_3_team04/reservation/domain/Reservation.kt

```diff
@@ -0,0 +1,186 @@
+package com.grepp.nbe1_3_team04.reservation.domain
+
+import com.grepp.nbe1_3_team04.global.domain.BaseEntity
+import com.grepp.nbe1_3_team04.member.domain.Member
+import jakarta.persistence.*
+import jakarta.validation.constraints.NotNull
+import org.hibernate.annotations.SQLDelete
+import java.time.LocalDateTime
+
+@SQLDelete(sql = "UPDATE reservation SET is_deleted = 'TRUE' WHERE reservation_id = ?")
+@Entity
+class Reservation private constructor(
+    court: Court,
+    member: Member,
+    team: Team,
+    matchDate: LocalDateTime,
+    reservationStatus: ReservationStatus,
+    gender: ParticipantGender
+) :
+    BaseEntity() {
```

Comment:
저도 잘 신경못쓰는거고 그냥 별건 아닌데 이렇게 하는게 더 예쁘지않을까요? ㅋㅋㅋㅠㅠ
```suggestion
class Reservation private constructor(
    court: Court,
    member: Member,
    team: Team,
    matchDate: LocalDateTime,
    reservationStatus: ReservationStatus,
    gender: ParticipantGender
) : BaseEntity() {
```

: 로 상속이나, 타입을 지정해줄때 줄바꿈이있으면 뭔가 좀 읽기어렵다해야되나.. 하여튼 별건 아니긴합니다

Reply:
빨리 하려다보니 신경을 못 쓴 부분이 많네요.. 확인하고 수정하겠습니다!